### PR TITLE
A couple of bug fixes for the ftp storages and added the use of "localtime()" to get the correct timestamp. 

### PR DESCRIPTION
**Pull request by ilCapo77**

### CODE REVIEW

File: dbbackup/dbcommands.py [hunk 1]
```diff
@@ -219,7 +219,7 @@ def filename(self, servername=None, wildcard=None):
         params = {
             'databasename': self.database['NAME'].replace("/", "_"),
             'servername': servername or settings.HOSTNAME,
-            'timestamp': timezone.now(),
+            'timestamp': timezone.localtime(timezone.now()),
```
Comment (ZuluPro) [inline dbbackup/dbcommands.py]: Is this line is really helpful, below the ``localtime``'s source:
```
Type:        function
String form: <function localtime at 0x7f50a51441b8>
File:        /usr/local/lib/python2.7/dist-packages/django/utils/timezone.py
Definition:  timezone.localtime(value, timezone=None)
Source:
def localtime(value, timezone=None):
    """
    Converts an aware datetime.datetime to local time.

    Local time is defined by the current time zone, unless another time zone
    is specified.
    """
    if timezone is None:
        timezone = get_current_timezone()
    # If `value` is naive, astimezone() will raise a ValueError,
    # so we don't need to perform a redundant check.
    value = value.astimezone(timezone)
    if hasattr(timezone, 'normalize'):
        # This method is available for pytz time zones.
        value = timezone.normalize(value)
    return value
```

If yes, the tests failed, could you fix it and make a bunch of tests for this line. Example:
- With ``USE_TZ = True``
- With ``USE_TZ = False``

**Comment by ZuluPro** [comment]
FTP and SFTP storages will be replaced by Django-Storage-Redux ones. Until they are not tested and changed, your contributions are very welcomed.

Can you rebase you commit to only one ? You made 3 commits for 1 line.

**Comment by ZuluPro** [comment]
You can use ``git rebase`` for this usage. See here: http://stackoverflow.com/questions/6934752/combining-multiple-commits-before-pushing-in-git#answer-6934882

**Comment by ilCapo77** [comment]
Ok. Thank you very much for your patience...

**Pull request closed without merging**